src/Microsoft.AspNetCore.Blazor/Json/SimpleJson/SimpleJson.cs

```diff
@@ -592,9 +593,16 @@ public static T DeserializeObject<T>(string json, IJsonSerializerStrategy jsonSe
 
         public static T DeserializeObject<T>(string json)
         {
+            CurrentJsonSerializerStrategy.SetPropertyNamingScheme(DefaultPropertyNamingScheme);
```

Comment:
Did this instead of recreating a new instance of ```PocoJsonSerializerStrategy``` class as I realized the class has a cache for future usage on the same type. So I did this to make sure that the CurrentJsonSerializationStrategy persists.

Reply:
This isn't thread safe. If two different threads are trying to use different serialization strategies, one of them will get the wrong result. I know this issue doesn't occur in browser-side code today, but it would be severe once we enable server-side rendering.

Does `SimpleJson` fundamentally rely on `CurrentJsonSerializerStrategy` being static? If so we might not be able to use this approach.

Reply:
Indeed you're right this will be a problem as it can lead to undesired results on both serializing and deserializing. 

As per investigation last time I reverted instantiating a new ```PocoJsonSerializerStrategy``` class as it used it for setting a Setter/Getter cache per type for future serialization/deserialization. So far that's the main reason it was done as a static property as per what I could see. 

I could disregard that cache altogether for every call and create a new instance, but I feel it might affect performance. I have some ideas to try to preserve the cache somehow per new instance. Will investigate a bit more on this.